.github/workflows/tests.yml

```diff
@@ -69,15 +81,17 @@ jobs:
           services: ${{ env.DOCKER_SERVICE }}
 
       - name: Start Redis stack image
+        id: stack_infra
         uses: hoverkraft-tech/compose-action@v2.0.1
-        if: ${{ matrix.redis == '7.4' }}
+        if: ${{ matrix.redis >= '6.2' && matrix.redis < '8.0' }}
         with:
           compose-file: .github/docker-compose.yml
           services: redis-stack
 
       - name: Start Redis cluster image
+        id: cluster_infra
         uses: hoverkraft-tech/compose-action@v2.0.1
-        if: ${{ matrix.redis == '7.4' }}
+        if: ${{ matrix.redis > '4.0' }}
```

Comment:
There are no options for Redis outside this condition. Should we keep it just if it is needed for the future or we can drop it?

Reply:
same as above